documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -1,29 +1,29 @@
 # Performance Tests Methodology
 
 ## Introduction
-This page describes methodology of Knotx application performance tests.
+This page describes the methodology of Knot.x performance tests.
 
 ## Goals
-Goals of conducted tests are as following:
-* Assess system performance,
+The goals of the tests conducted are as follows:
+* Assess system performance.
 * Assess system stability.
 
 ### Metrics
-During our tests we focus on following metrics and correlate it to came out with observations:
-* throughput,
-* response time - 90% Line,
-* quantity of Virtual Users,
-* Number of errors (HTTP response codes > 400, Knotx logs),
-* infrastructure resources (cpu, memory, disk, network),
-* application resources (jmx heap size, garbage collections).
+During our tests, we focus on the following metrics which, when correlated, allow us to come up with number of observations about the system:
+* Throughput.
+* Response time - 90% Line.
```

Comment:
I may be nitpicking a little but full stops usually follow bullet points that are full sentences :)
Most of these are just single phrases.